src/apiserver/client.go

```diff
@@ -40,19 +40,23 @@ func (c clientImpl) GetNodeInfo(nodeName string) (*NodeInfo, error) {
 	}
 
 	return &NodeInfo{
-		NodeName:    node.ObjectMeta.Name,
```

Comment:
There is a typo in commit message: `kubelete` :fr:

Reply:
I saw it but was too lazy to re-do it 😊